packages/web-react/src/types/dropdown.ts

```diff
@@ -1,5 +1,13 @@
 import { ComponentPropsWithRef, ElementType, LegacyRef, ReactNode } from 'react';
-import { Booleanish, ChildrenProps, ClickEvent, PlacementDictionaryType, StyleProps } from './shared';
+import {
+  AlignmentXExtendedDictionaryType,
+  AlignmentYExtendedDictionaryType,
+  Booleanish,
+  ChildrenProps,
+  ClickEvent,
+  PlacementDictionaryType,
+  StyleProps,
+} from './shared';
```

Comment:
Sigh: If Prettier didn't change formatting here, we could quickly see which imports are new… 🤷🏻‍♂️

Reply:
We can still change the line width to 80 chars :-) But base on 🔤 order, first two ;-)